pyvista/plotting/plotting.py

```diff
@@ -5041,26 +5041,51 @@ def add_lines(self, lines, color='w', width=5, label=None, name=None):
             If an actor of this name already exists in the rendering window, it
             will be replaced by the new actor.
 
+        connected : bool, default: False
+            Treat ``lines`` as points representing a series of *connected* lines.
+            For example, two connected line segments would be represented as
+            ``np.array([[0, 0, 0], [1, 0, 0], [1, 1, 0]])``. If ``False``, an *even*
+            number of points must be passed to ``lines``, and the lines need not be
+            connected.
+
+
         Returns
         -------
         vtk.vtkActor
             Lines actor.
 
         Examples
         --------
+        Plot two lines.
+
         >>> import numpy as np
         >>> import pyvista
         >>> pl = pyvista.Plotter()
         >>> points = np.array([[0, 1, 0], [1, 0, 0], [1, 1, 0], [2, 0, 0]])
-        >>> actor = pl.add_lines(points, color='yellow', width=3)
+        >>> actor = pl.add_lines(points, color='purple', width=3)
+        >>> pl.camera_position = 'xy'
+        >>> pl.show()
+
+        Adding lines with ``connected=True`` will add a series of connected
+        line segments.
+
+        >>> pl = pyvista.Plotter()
+        >>> points = np.array([[0, 1, 0], [1, 0, 0], [1, 1, 0], [2, 0, 0]])
```

Comment:
This is just nitpicking but I wouldn't redefine the same array here, emphasizing that it's the same data with just a different kwarg being plotted. But I don't insist if you prefer it this way.

Reply:
ah, I was thinking it would be useful to be see the data plotted directly above the plot itself rather than needing to scroll above the first plot. But I'm happy to remove it if anyone else thinks it would be clearer